read.php

```diff
@@ -67,16 +67,17 @@
     // var_dump($result);
     // exit();
 
-
-    // var_dump($result[0]["date"]);
-    // exit();
-    // <tr><td>deadline</td><td>todo</td><tr>の形になるようにforeachで順番に$outputへデータを追加
-    // `.=`は後ろに文字列を追加する，の意味
+    //DBから取得した値をそれぞれの変数煮詰めchart.jsで取得する
     foreach ($result as $record) {
```

Comment:
```suggestion
    foreach ($result as $record) {
        $day .= "{$record["date"]}','";
        $weights .= "{$record["weight"]}','";
        $bodyfats .= "{$record["bodyfat"]}','";
        $memos .= "<tr><td >メモ：{$record["memo"]} </td></tr>";
    }
```

分ける必要ないので一個にまとめてよさそうです〜！